atoma-proxy/src/server/handlers/completions.rs

```diff
@@ -151,8 +151,10 @@ pub async fn completions_create(
                     update_state_manager_fiat(
                         &state.state_manager_sender,
                         metadata.user_id,
-                        metadata.fiat_estimated_amount.unwrap_or_default(),
+                        metadata.max_total_num_compute_units as i64,
                         0,
+                        metadata.price_per_million.unwrap_or_default(),
```

Comment:
How come unwrap or default ? The price should never be 0, I believe..

Reply:
same as above